lib/ask-stack-result-view.coffee

```diff
@@ -198,8 +204,30 @@ class AskStackResultView extends ScrollView
           atom.workspace.activatePreviousPane()
           # editor = atom.workspace.activePaneItem
           editor = atom.workspace.getActivePaneItem()
-          editor.insertText(code)
 
+          # Check if an id was passed in to find information about the author and source
+          if id != undefined
```

Comment:
Refactored the previous `qa` code and changed this so that it all happens within an if statement here.